packages/pointer_interceptor/pointer_interceptor/README.md

```diff
@@ -44,26 +45,46 @@ All the cases above have in common that they attempt to render Flutter widgets *
 There's two ways that the `PointerInterceptor` widget can be used to solve the problems above:
 
 1. Wrapping your button element directly (FAB, Custom Play/Pause button...):
-
-    ```dart
-    PointerInterceptor(
-      child: ElevatedButton(...),
-    )
-    ```
+<?code-excerpt "main.dart (PointerInterceptorWrapper)"?>
+```dart
+PointerInterceptor(
+  child: ElevatedButton(
+    key: const Key('clickable-button'),
+    child: const Text('Works As Expected'),
+    onPressed: () {
+      _clickedOn('clickable-button');
+    },
```

Comment:
All this extra code makes the README example more complicated than what it needs to be.

You can remove unnecessary code from an excerpt, and render "..." in between the gaps, by doing something like this:

```
// #docregion PointerInterceptorWrapper
PointerInterceptor(
  child: ElevatedButton(
// #enddocregion PointerInterceptorWrapper
     key: const Key('clickable-button'),
     child: const Text('Works As Expected'),
     onPressed: () {
      _clickedOn('clickable-button');
    },
// #docregion PointerInterceptorWrapper
  ),
),
// #enddocregion PointerInterceptorWrapper
```

That way the example will still be based in the code, but it'll be more similar to the previous version!

---

The string used to "join" different parts of the example is called `plaster`, and can be configured with something like:

```
<?code-excerpt "main.dart (PointerInterceptorWrapper)" plaster="!!!"?>
```

The [default `plaster` value]() seems to be `···` (which I think is fine!):
 https://github.com/flutter/packages/blob/8163731c489338377154eb3fde2cfdd9293373c0/script/tool/lib/src/update_excerpts_command.dart#L154

---

(This applies to all generated excerpts in this file!)

Reply:
That's awesome. I will update this right away.

Reply:
Done.